liferay-echo/liferay-portal
# LPS-169805 Replace liferay liferay-ui:icon-menu with clay:dropdown-menu in page.jsp

Motivation
=======
In order to avoid futuro accessibility issues, we have decided to change liferay-ui:icon-menu with clay:dropdown-menu.

Proposed Solution
========
Use clay:dropdown-menu instead of liferay-ui:icon-menu.

Steps to Verify
========
1. Go to Web Content > create new Web Content
2. In the WB options, go to Related Assets > select an asset
3. We should be able to select assets and remove them 

Screenshots 
========
![despues](https://user-images.githubusercontent.com/91208451/204566234-aa4f6bd2-6557-423e-850c-107ec2206a42.gif)

CODE:
- I have renamed the function "getSelectorEntries" to "getActionDropdownItems" and it now returns a list of "DropdownItems". I have deleted the methods that are no longer used.
- For the propsTransformer, I have copied the code in page.jsp with some changes. I have changed "entityId" and "entryLink" to a HTML template syntax, so the code is more legible. 
Besides, I have passed one more attribute called "removeIcon" to add the icon when an item is selected.
